ci-operator/step-registry/rosa/aws/sts/provision/proxy/rosa-aws-sts-provision-proxy-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: rosa-aws-sts-provision-proxy
+  steps:
+  - ref: aws-provision-vpc-shared
+  - chain: aws-provision-bastionhost
+  - chain: rosa-aws-sts-provision
```

Comment:
To connect the proxy cluster, we need through the bastion machine. To be easily setup the environment with the proxy&bastion, we need to add the step `proxy-config-generate` after `rosa-aws-sts-provision` to generate the setup script.